src/components/Content/TaskManager/TaskForm/TaskForm.css

```diff
@@ -47,7 +47,7 @@
   min-width: 150px;
   border-radius: 0;
   border-color: #a5a5a5;
-}
+} */ */
```

Comment:
@Moe-Shoman (1) Should this all be commented out? (2) If so, should there just be one `*/`?

Reply:
Removed 👍